### ISSUE TITLE
Examples: install_allegro missing

### ISSUE THREAD

**Issue by baylej** [TheSos/allegrojs#20]
Some examples (exbounce, exgame, stress, offstress) don't call `install_allegro` (or `allegro_init`) before using allegro.js.

Is `install_allegro` really mandatory?

**Comment by TheSos** [comment]
Nope, I should probably get rid of that :P Kept it in case something will have to be done before doing anything else, but it's actually empty!

**Comment by ratkingsminion** [comment]
It's not empty?
